src/plugins/console/public/application/containers/editor/legacy/console_editor/editor_output.tsx

```diff
@@ -86,15 +144,17 @@ function EditorOutputUI() {
         data
           .map((result) => {
             const { value, contentType } = result.response;
+
+            let editorOutput;
             if (readOnlySettings.tripleQuotes && isJSONContentType(contentType)) {
-              return safeExpandLiteralStrings(value as string);
+              editorOutput = safeExpandLiteralStrings(value as string);
             }
+
             if (isMapboxVectorTile(contentType)) {
```

Comment:
If the response comes back with errors, we only see `[object ArrayBuffer]` in the response. I think it would be helpful to return the JSON response if the query returns errors.

An example of a request returning an error might be a simple typo like this:
```
GET kibana_sample_data_flights/_mvt/DestLocation/0/0/0
{
  "sze": 10
}
```

Reply:
Since we are expecting a `responseType` as `ArrayBuffer` there, we can only display a generic message if something went wrong like [es.ts#L68](https://github.com/elastic/kibana/blob/e7a7f3a0127be1b9175ab124114e90a70c40c719/src/plugins/console/public/lib/es/es.ts#L68) The specific message from the server cannot be parsed, the `responseType` must be as `text` in that case.